osutil/cmp.go

```diff
@@ -58,10 +58,12 @@ func FilesAreEqual(a, b string) bool {
 		return false
 	}
 
-	return streamsEqual(fa, fb)
+	return FileStreamsEqual(fa, fb)
 }
 
-func streamsEqual(fa, fb io.Reader) bool {
+// FileStreamsEqual compares two file streams and returns true if both
+// have the same content.
+func FileStreamsEqual(fa, fb io.Reader) bool {
```

Comment:
Those are really streams rather than files, or Readers if we want to be more precise. Files are *os.File.

This also seems somewhat unrelated to the operating system, but we can easily reorganize later. Let's please just fix the name since we changed it in this PR.